source/common/upstream/cluster_manager_impl.cc

```diff
@@ -384,6 +384,13 @@ ClusterManagerImpl::ClusterManagerImpl(
             validation_context.dynamicValidationVisitor(), server,
             dyn_resources.ads_config().config_validators());
 
+    JitteredExponentialBackOffStrategyPtr backoff_strategy;
+    auto& grpc_services = dyn_resources.ads_config().grpc_services();
+    if (!grpc_services.empty() && grpc_services[0].has_envoy_grpc()) {
+      backoff_strategy = Config::Utility::prepareJitteredExponentialBackOffStrategy(
+          grpc_services[0].envoy_grpc(), random_, absl::nullopt, absl::nullopt);
+    }
```

Comment:
Looks like the function that I commented above.
In this case I think it should be a static function in the subscription-factory.

Reply:
would it be better if this is moved to the `utility` class as a new method `prepareJitteredExponentialBackOffStrategy(const envoy::config::core::v3::ApiConfigSource& api_config_source, Random::RandomGenerator& random)` instead?

Reply:
done, thanks!